File: backend/dialogue.py
# Generates dialogue prompts

import random

@app.route('/dialogue')
def generate_dialogue_prompt():
    prompts = [
        "Write a conversation between two old friends meeting after many years.",
        "Create a dialogue between a customer and a cashier at a grocery store.",
        "Imagine a conversation between a detective and a suspect during an interrogation."
        # I will add  more dialogue prompts here
    ]
    return random.choice(prompts)

File: backend/dialogue.js
// backend/dialogue.js

const dialoguePrompts = [
  "Write a dialogue between two characters who just met on a blind date.",
  "Write a conversation where one person is trying to convince another to take a huge risk.",
  "Create a dialogue between a parent and child about an important life lesson.",
  "Imagine a dialogue between two old friends who haven't seen each other in years.",
  "Write a dialogue where a character reveals a big secret."
];

function getRandomDialoguePrompt() {
  const randomIndex = Math.floor(Math.random() * dialoguePrompts.length);
  return dialoguePrompts[randomIndex];
}

module.exports = getRandomDialoguePrompt;
